tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -29,31 +29,77 @@
   retries: 60
   delay: 2
 
-- name: get podified OVN NB ovsdb-server IP
+- name: get podified OVN NB ovsdb-server service cluster IP
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc get pod ovsdbserver-nb-0 -o jsonpath='{.metadata.annotations.k8s\.v1\.cni\.cncf\.io/network-status}' | \
-    jq 'map(. | select(.name=="openstack/internalapi"))[0].ips[0]' | \
-    tr -d '"'
+    oc get svc --selector "statefulset.kubernetes.io/pod-name=ovsdbserver-nb-0" -ojsonpath='{.items[0].spec.clusterIP}'
```

Comment:
This is failing in CI because it seems that it takes a few seconds to assing this IP to the pod and it's not ready when it queries it. I tested it in a held node  adding a few retries like in the task above and it worked after a couple of attempts

Reply:
this is weird as we wait for the pod's status.phase==Running